docs/manual_service_startup.md

```diff
@@ -85,7 +85,7 @@ From the root dir of `elixir-omg`:
 mix compile
 mix run --no-start -e \
  '
-   contents = OMG.Eth.DevHelpers.prepare_env!() |> OMG.Eth.DevHelpers.create_conf_file()
+   contents = Support.DevHelper.prepare_env!() |> Support.DevHelper.create_conf_file()
```

Comment:
this document could be removed completely, it's not relevant anymore.

Reply:
It's not true.  Also demos are based on manual startup.